app/controllers/sessions.rb

```diff
@@ -0,0 +1,25 @@
+# LOG IN PAGE
+get '/sessions/new' do
+  erb :'sessions/login'
+end
+
+# LOG IN POST
+post '/sessions' do
+  user = User.find_by(email: params[:email])
+  if user && user.authenticate(params[:password])
+    session[:user_id] = user.id
+    puts "#{user.username} logged in. Redirecting."
+    redirect '/'
```

Comment:
if the index redirects to '/questions', then wouldn't it just make sense to redirect '/questions' upon login? OR should we redirect the user to their user dashboard (/users/:id) upon successful login?